h2o-py/tests/testdir_algos/glm/pyunit_GH_6722_constraints_on_collinear_cols2.py

```diff
@@ -0,0 +1,56 @@
+import h2o
+from h2o.estimators.glm import H2OGeneralizedLinearEstimator
+from tests import pyunit_utils
+
```

Comment:
```suggestion


```
Ideally, I would try to stick to PEP-8 and [keep 2 blank lines between top-level functions](https://peps.python.org/pep-0008/#blank-lines).